src/select-options.ts

```diff
@@ -10,6 +10,7 @@ function selectOptionsBase(
   values: HTMLElement | HTMLElement[] | string[] | string,
   init?: MouseEventInit,
 ) {
+  if (!hasPointerEvents(select)) return
```

Comment:
`selectOptions()` on `<select/>` is possible without pointer events. We don't want to break backward compatibility here.
Please apply it only to the pointer events.

Reply:
Alright. If I understand correctly then we don't want to prevent events to propagate here. Rather I should revert changes from `select-options.ts` file. Am I right?

Reply:
I think for now the solution is to skip the pointer events in [line 49ff](https://github.com/testing-library/user-event/blob/c2186493bb329b6e50fe4256df7921377e3a309a/src/select-options.ts#L49-L92) if the clicked element (select/option) should not have pointer events but keep the call to `selectOption()`

Reply:
I think this is what we wanted here to maintain behaviour,
https://github.com/testing-library/user-event/pull/631/files#diff-1031ae1312b9a49cb4a49f43fdd91f4d13537c7f6f31a7e3021d4ef7797a237aR48

Reply:
See [line63](https://github.com/testing-library/user-event/blob/a72cc3f64c2b9c20808d3d5e2f7e64870d07cc34/src/select-options.ts#L63) and [line70](https://github.com/testing-library/user-event/blob/a72cc3f64c2b9c20808d3d5e2f7e64870d07cc34/src/select-options.ts#L70) the call to `selectOption` .
Those calls should not depend on pointer events.

The issue for another day might be to add keyboard events doing this selections. But for now I think it's sensible to just skip all pointer events and only select the options the user wanted to select.